h2o-core/src/main/java/water/rapids/Rapids.java

```diff
@@ -119,6 +119,9 @@ public static Val exec(String rapids, Session session) {
   // Set of characters that may appear in a number. Note that "NaN" or "nan" is also a number.
   private static Set<Character> validNumberCharacters = StringUtils.toCharacterSet("0123456789.-+eEnNaA");
 
+  private static Set<Character> validBooleanCharacters = StringUtils.toCharacterSet("FALSEfalseTRUEtrue");
+
+  private static Set<Character> validInitBooleanCharacters = StringUtils.toCharacterSet("FfTt");
```

Comment:
you would not need this if direction was given by +1, -1

Reply:
Deleted.